code/modules/mob/living/carbon/human/life/handle_regular_hud_updates.dm

```diff
@@ -2,150 +2,191 @@
 
 /mob/living/carbon/human/handle_regular_hud_updates()
 	. = ..()
-	if(!(.))
+	if(!.)
 		return FALSE
 
-	if(stat != DEAD) //the dead get zero fullscreens
+	update_sight()
+
+	if(stat == UNCONSCIOUS && health <= get_crit_threshold())
+		var/severity = 0
```

Comment:
```suggestion
		var/severity = CLAMP(FLOOR((health * -1) / 10), 0, 10)
```

Would that save you from the large switch case?

Reply:
That code was just moved and not changed, if you wanna do something that's not 1 line you should PR it

Reply:
do the change

Reply:
The suggestion is good, but we don't demand of contributors that they refactor everything they touch on big code moves or mass-replacements.

Reply:
You just gotta click the button to commit the suggestion

Reply:
That would require a bit more than that, this is not a one-line change. (if I wanted to make full use of it that is)